### DISCUSSION TITLE
在Windows编译2.11.0经验

### DISCUSSION THREAD

**Discussion by xbctmri** [MetaCubeX/ClashMetaForAndroid#340] [General]
最近发现这个项目有两处改动，垃圾收集从UseParallelGC换成了UseZGC，以及Meta（稳定版）与Alpha（测试版）构建任务分开，导致read.me的编译方法不好用了。
于是我查看openjdk wiki，发现UseZGC在Windows至少要jdk 14（实验性质）及jdk 15。于是我尝试把jdk 11换成了**jdk 15**，

然后把最后的编译命令换成

./gradlew --no-daemon app:assembleAlphaRelease
./gradlew --no-daemon app:assembleMetaRelease

然后就编译出来了。
以上经验，仅供参考

[gradle.properties](https://github.com/MetaCubeX/ClashMetaForAndroid/commit/43337eacd2859f5661f0636ec8ff44d8b4ada827#diff-3d103fc7c312a3e136f88e81cef592424b8af2464c468116545c4d22d6edcf19)

[.github/workflows/build-debug.yaml](https://github.com/MetaCubeX/ClashMetaForAndroid/commit/43337eacd2859f5661f0636ec8ff44d8b4ada827#diff-ce28e231fbab59519d5f46f6f9976f9e477fe35e68ab1905de972f37da9159ca)
Copied!

**Comment by djwjb** [comment]
![mm_20240926093630](https://github.com/user-attachments/assets/f81d55a3-e216-473b-af7f-5065214d8607)
直接用jbr，17，
看workflows里面的配置，所以用  `--no-daemon app:assembleMetaRelease`

**Reply by 5lonely** [reply]
牛逼，大佬怎么才能在全局模式下显示测速

**Comment by p0we7** [comment]
我自己生成了 keystore ，用 actions 全新编译。无论是 debug 和 release 编译出来的 apk 安装都提示 “package is null” 知道是什么问题吗？

**Reply by 5lonely** [reply]
大佬电信免流全局不能测速怎么回事呀
